bin/templates/cordova/lib/plugin-build.gradle

```diff
@@ -20,8 +20,8 @@
 
 buildscript {
     repositories {
+        mavenCentral()
         google()
```

Comment:
I think there have been problems in the past with mavenCentral not finding google dependencies and it's preferred to have google() on top

Reply:
These changes are apart of the https://github.com/apache/cordova-android/pull/1251 PR, I went ahead and corrected the order on that PR and rebased this PR.